akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,65 @@
+jQuery(document).ready(function ($) {
+  initOldVersionWarnings($);
+});
+
+function initOldVersionWarnings($) {
+  $.get("http://doc.akka.io/docs/versions.json", function (akkaVersionsData) {
+    var thisVersion = browsedAkkaVersion();
+    if (thisVersion.includes("-SNAPSHOT")) {
+      console.log("Detected SNAPSHOT Akka version...");
+      // we could show a 'hey, this is a snapshot' notice
+    } else {
+      for (var series in akkaVersionsData) {
+        if (browsedAkkaVersion().startsWith(series)) {
+          showVersionWarning(thisVersion, akkaVersionsData[series]);
+        }
+      }
+    }
+  });
+}
+
+
+function showVersionWarning(version, seriesInfo) {
+  console.log("Akka version:", version);
+  var targetUrl = ("" + window.location).replace(version, seriesInfo.latest);
+  var $floatyWarning = $('<div id="floaty-warning"/>');
+
+  $floatyWarning
+      .html('' +
+          'You are browsing the docs for Akka ' + version + ', ' +
+          'however the latest release in this series is: ' +
+          '<strong><a href="' + targetUrl + '">' + seriesInfo.latest + '</a></strong>. <br/>' +
```

Comment:
Minor style detail: It's a bit inconsistent with the property accesses, here it's `seriesInfo.latest` and below `seriesInfo['instead']`, I would use the former unless having a variable selecting the property or something like that.